### PR TITLE
ready_to_deploy our "mvp base" with bumped and updated front: 

### LINKED ISSUE #80: bump packages and react to 18-version and adapt to Typescript, Vite

### PR DESCRIPTION
**Pull request by miniMax18**
Vite, TypeScript, Docker, with Readme to up front-end app by Dockerfile(docker use 'npm run start') and localhost(in cli prompt 'npm run dev') option to launch in 3000 port.

### CODE REVIEW

File: Dockerfile_18_vite_alpine_4
Comment (szmktk) [inline Dockerfile_18_vite_alpine_4:1]: Which `Dockerfile` do we want to retain?

File: README_vite.md
Comment (szmktk) [inline README_vite.md:1]: Why is there a separate `README` for Vite? Let's create a section in the main README.

**Review by szmktk — APPROVED** [review]

File: README_vite.md
Comment (miniMax18) [reply README_vite.md:1]: Probably Vite generate this one file in init. Ok, this is to do, I should do it with next commits and attach to this PR with other code fix.

File: Dockerfile_18_vite_alpine_4
Comment (miniMax18) [reply Dockerfile_18_vite_alpine_4:1]: Dockerfile is main, another is for info, that my point of view on this. 2 files are not too many. If that break radically a convention We can to delete one. In my opinion is not that What we should focus- but that my opinion. To delete - is folder with dockerfiles in first(I added this folder to .gitignore but committed it- probably squash or git reset? did it).  Another file to delete with postfix ***123 like package-lock123.json.

**Review by bartmkraus — APPROVED** [review]

**Pull request merged**